templates/shared_services/certs/Dockerfile.tmpl

```diff
@@ -12,7 +12,7 @@ RUN apt-get update \
     && curl -sL https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor | tee /etc/apt/trusted.gpg.d/microsoft.gpg > /dev/null \
     && AZ_REPO=$(lsb_release -cs) \
     && echo "deb [arch=amd64] https://packages.microsoft.com/repos/azure-cli/ $AZ_REPO main" | tee /etc/apt/sources.list.d/azure-cli.list \
-    && apt-get update && apt-get -y --no-install-recommends install azure-cli \
+    && apt-get update && apt-get -y install azure-cli="2.36.0-1~stretch" \
```

Comment:
Is this needed for this change? Why pin on this version?

Reply:
Not really needed. I just used the same version as in the other bundles, this was just helping to build the docker images locally quicker (for running tests) as when they are not pinned, this becomes a new layer each time azure-cli is updated, and this install step can take a really long time on my machine (>30 minutes), because of this: https://github.com/microsoft/WSL/issues/2775 

I'm happy to remove this though.